web_development_101/project_html_css.md

```diff
@@ -1420,6 +1420,8 @@ Submit a link below to the github repo with your files in it by using a pull req
 * [Douglasbsx Solution (Easy)](https://github.com/douglasbsx/Google-homepage) - [View in Browser](https://douglasbsx.github.io/Google-homepage/)
 * [Ahmad Javed's Solution (Easy)](https://github.com/ahmad-codes/google-homepage) - [View in Browser](https://ahmad-codes.github.io/google-homepage/)
 * [jstnlester's Solution (Easy)](https://github.com/jstnlester/google-homepage) - [View in Browser](https://jstnlester.github.io/google-homepage/)
+BreatheCodes's easy solution
```

Comment:
Thank you for your submission! Could you follow the same format as the projects above? Thank you!